docs/layers/lang/ruby.md

```diff
@@ -46,8 +46,21 @@ gem install rubocop
 
 ### Inferior REPL process
 
-Start a `irb` inferior REPL process with `SPC l s i`. 
+Start a `irb` inferior REPL process with `SPC l s i`. You may change the REPL command by layer option `repl_command`. For example, if you want to use `pry`, load this layer via:
 
```

Comment:
the code block is toml, so it should  be `` ```toml  ``

Reply:
👍 fixed and re-pushed